server/tests/taxonomy_term.rs

```diff
@@ -219,3 +217,68 @@ mod move_mutation {
         .await;
     }
 }
+
+#[cfg(test)]
+mod create_mutation {
+    use test_utils::*;
+
+    #[actix_rt::test]
+    async fn creates_new_taxonomy_term() {
+        for taxonomy_type in TAXONOMY_TYPES_WITHOUT_ROOT.iter() {
```

Comment:
```suggestion
        for taxonomy_type in ["topic", "topicFolder"].iter() {
```

That's enough.

Reply:
since we have shortened the `TAXONOMY_TYPES_WITHOUT_ROOT`` (now called ALLOWED_TAXONOMY_TYPES_CREATE) there is no need to change this line